docs/neutral.rst

```diff
@@ -0,0 +1,60 @@
+Cloud agnostic computing
```

Comment:
This fille will be enhanced shortly with instructions on running a cloud pure k8s use case for cloud agnostic deployments